### ISSUE TITLE
Go memstats no longer reported in 1.0

### ISSUE THREAD

**Issue by francisdb** [influxdata/influxdb#7272]
Since this is not mentioned in the changelog I guess this is a bug?

Influxdb 0.13
```bash
> curl -s http://127.0.0.1:8086/debug/vars | grep memstats
"memstats": {"Alloc":240914768,"TotalAlloc":5988867901232,"Sys":8533 ...
>
```

Influxdb 1.0
```bash
> curl -s http://127.0.0.1:8086/debug/vars | grep memstats
>
```

**Comment by jsternberg** [comment]
It looks like it is not under `"runtime"`. I'm investigating to see if that's where it's supposed to be or if it got put into the wrong section when we refactored expvars.

**Comment by jwilder** [comment]
I see them under `runtime` in `1.0`.

```
$ curl -s localhost:8086/debug/vars | grep runtime
"runtime": {"name":"runtime","tags":{},"values":{"Alloc":6187664,"Frees":72753,"HeapAlloc":6187664,"HeapIdle":5644288,"HeapInUse":8249344,"HeapObjects":19503,"HeapReleased":0,"HeapSys":13893632,"Lookups":41,"Mallocs":92256,"NumGC":5,"NumGoroutine":28,"PauseTotalNs":812000,"Sys":18757880,"TotalAlloc":20621504}},
```
`memstats` and `cmdline` are builtin expvar keys which were probably implicitly added in `0.13`.

**Comment by jsternberg** [comment]
Fixed via #7277.
